### PR TITLE
<StatisticsWidget />- add short text when title doesn't fit

### PR DESCRIPTION
**Pull request by maxvektor**
### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by maxvektor** [comment]
@mosheKerbel 
Take a look, should be fine now.

File: src/StatisticsWidget/test/StatisticsWidget.spec.js [hunk 1]
```diff
@@ -34,20 +34,31 @@ describe('StatisticsWidget', () => {
...
+        const count = await driver.getItemsCount();
 
-      expect(count).toBe(1);
-    });
+        expect(count).toBe(1);
+      });
 
-    it('should render title based on props', async () => {
-      data.statistics[0].title = 'Changed title';
+      it('should render title based on props', async () => {
```
Comment (mosheKerbel) [inline src/StatisticsWidget/test/StatisticsWidget.spec.js]: also expect no short title to exist

File: src/StatisticsWidget/test/StatisticsWidget.spec.js [hunk 2]
```diff
@@ -34,20 +34,31 @@ describe('StatisticsWidget', () => {
...
-      const { driver } = render(<StatisticsWidget {...data} />);
-      const title = await driver.getStatisticTitle(0);
+        const { driver } = render(<StatisticsWidget {...data} />);
+        const title = await driver.getStatisticTitle(0);
 
-      expect(title).toBe('Changed title');
+        expect(title).toBe('Changed title');
+      });
+
+      it('should render short version', async () => {
```
Comment (mosheKerbel) [inline src/StatisticsWidget/test/StatisticsWidget.spec.js]: also expect to have the full title

**Comment by maxvektor** [comment]
I still have failed test but looks like it is not connected to this PR.

**Comment by maxvektor** [comment]
again failed because of flaky tests.

**Comment by mosheKerbel** [comment]
i merged the other one
it caused some conflicts to fix here...
@mykhailou

**Comment by maxvektor** [comment]
@LittleTal Hey! 
Could you review this one?

https://wix-wix-style-react-pr-3909.surge.sh/?selectedKind=WIP&selectedStory=StatisticsWidget&full=0&addons=0&stories=1&panelRight=0

`Custom text instead of ellipsis` section

**Comment by mosheKerbel** [comment]
> @LittleTal Hey!
> Could you review this one?
> 
> https://wix-wix-style-react-pr-3909.surge.sh/?selectedKind=WIP&selectedStory=StatisticsWidget&full=0&addons=0&stories=1&panelRight=0
> 
> `Custom text instead of ellipsis` section

@mykhailou why did you ask review only for this section?
i thought that the other 3 on bottom were also ready for review

**Comment by maxvektor** [comment]
@LittleTal
I have new version. Now should work fine

**Comment by mosheKerbel** [comment]
flaky unrelated tests
merged.

**Pull request merged**